gammapy/conftest.py

```diff
@@ -12,6 +12,10 @@
 ## from the list of packages for which version numbers are displayed
 ## when running the tests ... this was added in Astropy 1.0
 try:
+    # Removing h5py for now to get rid of this travis-ci fail
+    # https://travis-ci.org/gammapy/gammapy/jobs/86565582#L1074
+    del PYTEST_HEADER_MODULES['h5py']
```

Comment:
A simple `conda uninstall h5py` broke the builds where `h5py` wasn't installed, because then the uninstall commands returns a non-zero exit code. Not sure how to put this in an if statement to make it work for all builds.
@astrofrog Any advice how to achieve this in the bash script?

Another thing I tried was to remove `h5py` from the conda install command for optional dependencies.
Unfortunately it seems it still gets installed for some reason:
https://travis-ci.org/gammapy/gammapy/jobs/86619175#L1080
It's a bit frustrating that the executed commands don't show up in the travis-ci logs.
That would help figuring out what's going on.
@astrofrog Any idea how to achieve that?